python/src/trezorlib/toif.py

```diff
@@ -1,15 +1,19 @@
 import struct
 import zlib
+from dataclasses import dataclass
 from typing import Sequence, Tuple
 
-import attr
-
 from . import firmware
 
 try:
-    from PIL import Image
+    # This double import needed to properly type-check
+    # https://stackoverflow.com/questions/58236138/pil-and-python-static-typing/58236618#58236618
+    from PIL.Image import Image
+    from PIL import Image as img
```

Comment:
`from PIL import Image as pil` please, so that it's clearer in the usage/s below

also then we can presumably use `pil.Image` as the type hint, without double import

Reply:
Cool, the `pil.Image` then works fine. That brought me to actually do just `from PIL import Image` and then when typing the arguments and return values do `Image.Image`, which lets us to use `Image.new(...)` etc.

Reply:
Done in 4cfe54a